articles/140_topic_and_service_name_mapping.md

```diff
@@ -265,7 +335,7 @@ Here are some examples of translations between ROS topic names and the correspon
 | `/_foo/_bar`      | Topic     | `rt___foo___bar`       |
 | `/_foo/_bar/_baz` | Topic     | `rt___foo___bar___baz` |
 
-### ROS Topic and Service Name Length Limit
+#### ROS Topic and Service Name Length Limit
```

Comment:
The limit was effected by the fact that there was a prefix, but even if we drop the prefix, there is a length limit. It doesn't go away or stay based on the ros prefix.